ui/scss/component/_file-thumbnail.scss

```diff
@@ -1,13 +1,13 @@
 .file-thumbnail__viewed-bar {
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  width: 100%;
-  height: 5px;
-  background-color: var(--color-gray-7);
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 10px; // UPDATE: Changed thumbnail viewed bar height to 10px
+    background-color: var(--color-gray-7);
 }
 
 .file-thumbnail__viewed-bar-progress {
-  height: 5px;
-  background-color: var(--color-primary);
+    height: 10px; // UPDATE: Changed thumbnail viewed progress bar height to 10px
```

Comment:
Not sure about this. Besides, 100% would be fine since it's inside file-thumnail-viewed-bar.

Reply:
I had initially meant to remove these changes. I forgot. They are only for the version I have compiled for myself. I have no issues with retaining the original styling. Same with below.